src/execution/operator/csv_scanner/scanner/string_value_scanner.cpp

```diff
@@ -209,8 +211,20 @@ void StringValueResult::AddValueToVector(const char *value_ptr, const idx_t size
 		          TimestampCastResult::SUCCESS;
 		break;
 	}
-	default:
+	default: {
+		// By default we add a string
+		if (!Utf8Proc::IsValid(value_ptr, UnsafeNumericCast<uint32_t>(size))) {
```

Comment:
We only need to do this check if the target type is `VARCHAR` - perhaps we can use a separate case statement for this?